iothub_client/tests/iothubclient_ll_ut/iothubclient_ll_ut.c

```diff
@@ -3700,6 +3710,54 @@ TEST_FUNCTION(IoTHubClient_LL_UploadToBlob_with_NULL_source_and_size_greater_tha
     ///cleanup
     IoTHubClient_LL_Destroy(h);
 }
+
```

Comment:
Are there requirement strings to be written for each of these tests? You know, the // Tests_SRS_...

Reply:
I should have seen this...